universe/jupyterlab/config.json

```diff
@@ -162,64 +162,36 @@
     },
     "storage": {
       "type": "object",
-      "title": "Mesosphere Jupyter Service Storage Configuration",
-      "description": "Mesosphere Jupyter Service Storage Configuration",
+      "description": "DC/OS JupyterLab storage configuration.",
```

Comment:
Don't change the title and description unless needed. It will cause unnecessary conflicts in future rebases.